### ISSUE TITLE
Accessibility bug: screenreaders should announce disabled options as "dimmed" or "disabled"

### ISSUE THREAD

**Issue by a11y1** [JedWatson/react-select#5135]
How to reproduce the bug: 

- Go to https://react-select.com/homehttps://react-select.com/home

- Check "disabled" for the first example, "Single"

- Run the screenreader (I used Mac VoiceOver). The selected item is announced, but the "dimmed" or "disabled" description is missing. This can be seen using arrow key navigation (VO + arrow keys) or with auto read (VO + A).

Screenshot: 

<img width="617" alt="Screen Shot 2022-04-04 at 9 16 56 AM" src="https://user-images.githubusercontent.com/102971546/161567772-3b80306a-8b5c-489d-b344-fe5f7ca1eca8.png">

**Comment by SairamAlagapan** [comment]
Any update on this please?

**Comment by SairamAlagapan** [comment]
@lukebennett88 @Ke1sy - Is this achievable?
